e2e/_suites/fleet/fleet.go

```diff
@@ -1315,12 +1329,14 @@ func (fts *FleetTestSuite) theMetricsInTheDataStream(name string, set string) er
 					"dataset": "system." + set,
 					"enabled": "true",
 					"type":    name,
+					"os":      os,
 				}).Info("The " + name + "with value system." + set + " in the metrics")
 
 				if int64(int64(item.Path("last_activity_ms").Data().(float64))) > startTime {
-					log.WithField(
-						"Activity Time stamp for the "+name+"system."+name, "Is valid",
-					).Info("The " + name + "with value system." + set + " in the metrics")
+					log.WithFields(log.Fields{
+						"Activity Time stamp for the " + name + "system." + name: "Is valid",
```

Comment:
These should just be field identifiers like `log.Fields{"timestamp": timestamp}` But I must admit I'm not quite sure what this line is supposed to identify?

Reply:
I identify last activity time. I get time before and after  to get the difference

Reply:
Done